applications/cvcuda_basic/README.md

```diff
@@ -9,59 +9,69 @@ specification](https://dmlc.github.io/dlpack/latest/python_spec.html) in both CV
 We provide two [operators](../../operators/cvcuda_holoscan_interop/README.md) to handle the
 interoperability between CVCUDA and Holoscan tensors.
 
-# Using the docker file
+# Quick Start with HoloHub CLI Container
+
+Run the following command to build and run the CV-CUDA sample application in a Docker container:
+
+```bash
+./holohub run cvcuda_basic
+```
+
+# Advanced Build Steps
+
+## Using the docker file
 
 This application requires a compiled version of [CV-CUDA](https://github.com/CVCUDA/CV-CUDA).
 For simplicity a DockerFile is available. To generate the container run:
 
 ```bash
-./dev_container build --docker_file ./applications/cvcuda_basic/Dockerfile
+./holohub build-container cvcuda_basic
 ```
 
-The C++ version of the application can then be built by launching this container and using the provided `run` script.
+The C++ version of the application can then be built by launching this container and using the provided `holohub` CLI.
 
 ```bash
-./dev_container launch
-./run build cvcuda_basic
+./holohub run-container cvcuda_basic
+>>> ./holohub build cvcuda_basic
```

Comment:
Is there any reason for adding `>>>`?

Reply:
Intention was to show that the commands under `>>>` run in the docker container, but if it's not clear then let's remove `>>>`.